Sources/TokamakCore/StackReconciler.swift

```diff
@@ -70,61 +95,71 @@ public final class StackReconciler<R: Renderer> {
   private func setupState(
     id: Int,
     for property: PropertyInfo,
-    of compositeView: MountedCompositeView<R>
+    of compositeElement: MountedCompositeElement<R>,
+    body bodyKeypath: ReferenceWritableKeyPath<MountedCompositeElement<R>, Any>
   ) {
     // swiftlint:disable force_try
     // `ValueStorage` property already filtered out, so safe to assume the value's type
     // swiftlint:disable:next force_cast
-    var state = try! property.get(from: compositeView.view.view) as! ValueStorage
+    var state = try! property.get(from: compositeElement[keyPath: bodyKeypath]) as! ValueStorage
 
-    if compositeView.state.count == id {
-      compositeView.state.append(state.anyInitialValue)
+    if compositeElement.state.count == id {
+      compositeElement.state.append(state.anyInitialValue)
     }
 
     if state.getter == nil || state.setter == nil {
-      state.getter = { compositeView.state[id] }
+      state.getter = { compositeElement.state[id] }
 
       // Avoiding an indirect reference cycle here: this closure can be
       // owned by callbacks owned by view's target, which is strongly referenced
       // by the reconciler.
-      state.setter = { [weak self, weak compositeView] newValue in
-        guard let view = compositeView else { return }
-        self?.queueStateUpdate(for: view, id: id) { $0 = newValue }
+      state.setter = { [weak self, weak compositeElement] newValue in
+        guard let element = compositeElement else { return }
+        self?.queueStateUpdate(for: element, id: id) { $0 = newValue }
       }
     }
-    try! property.set(value: state, on: &compositeView.view.view)
+    try! property.set(value: state, on: &compositeElement[keyPath: bodyKeypath])
   }
 
   private func setupSubscription(
     for property: PropertyInfo,
-    of compositeView: MountedCompositeView<R>
+    of compositeElement: MountedCompositeElement<R>,
+    body bodyKeypath: KeyPath<MountedCompositeElement<R>, Any>
   ) {
     // `ObservedProperty` property already filtered out, so safe to assume the value's type
     // swiftlint:disable:next force_cast
-    let observed = try! property.get(from: compositeView.view.view) as! ObservedProperty
+    let observed = try! property.get(from: compositeElement[keyPath: bodyKeypath]) as! ObservedProperty
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/StackReconciler.swift;
  Line: 131;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 103 characters (`line_length`)